src/containers/shared/utils.js

```diff
@@ -151,7 +151,10 @@ export const getLocalizedCurrencySymbol = (lang = 'en-US', currency = 'USD') =>
   return formatted.split('1')[0].trim();
 };
 
-export const formatLargeNumber = (d = 0, digits = 4) => {
+export const formatLargeNumber = (d = 0, digits = 6) => {
```

Comment:
This method is used in a lot of different places across the explorer, are you sure those places aren't affected by your changes?

Reply:
It's only for the Tokens page and it works perfectly.

Reply:
It's also used in the token page header (the per-token page)